src/user-guide/task-implementation/job-scripts.rst

```diff
@@ -249,3 +249,23 @@ task to repeatedly poll for the results of the detached processes:
            # job.done indicator file does not exist yet.
            script = "[[ ! -f $RUN_DIR/job.done ]] && exit 1"
            execution retry delays = 10 * PT1M
+
+
+.. _PipeInJobScripts:
+
+Use of pipes in job scripts
```

Comment:
```suggestion
Use of Pipes in Job Scripts
```

Case change to keep in line with other headings.

Reply:
Applied.